packages/web-components/src/label/README.md

```diff
@@ -0,0 +1,122 @@
+# Label
+
+> A label represents a caption for an item in a user interface.
+
+<br />
+
+## **Design Spec**
+
+[Link to Label Design Spec in Figma](https://www.figma.com/file/jpWO2FMBefirTyThf5Rg2P/Label?node-id=2%3A476&t=QCdofuTbXkUjMS4d-0)
+
+<br />
+
+## **Engineering Spec**
+
+Due to the cost of associating elements across the shadow DOM boundary the `fluent-label` will not have access to the element it is intended to reference. For this reason the `fluent-label` is for visual purposes only.
+
+The fluent-label has several visual font size (small, medium, large) and font weight(regular, semibold) options. The fluent-label also provides appearances for required and disabled states.
+
+### Use Case
+
+Creating a simple label element with an optional info icon and optional required state
+
+<br />
+
+## Class: `Label`
+
+<br />
+
+### **Variables**
+
+<br />
+
+### **Fields**
+
+| Name       | Privacy | Type                           | Default     | Description                          |
+| ---------- | ------- | ------------------------------ | ----------- | ------------------------------------ |
+| `id`       | public  | `string`                       |             | Specifies the id of label            |
```

Comment:
I thought I commented here, but does `id` need to be observable? It's a standard HTML attribute and unless we need to observe/react to changes I don't think we need to call it out here, we can just add and associate like any other element.